cmd/introspect/main_test.go

```
@@ -1,12 +1,274 @@
+				"C":       "openai",
+				"D":       "openai",
+				"E":       "openai",
+			},
+			fallbackChains: map[string][]string{
+				"Outer":   {"Mid", "E"},
+				"Mid":     {"Inner", "D"},
+				"Inner":   {"InnerRR", "C"},
+				"InnerRR": {"A", "B"},
+			},
+		}
+		var cap captureLog
```

golangci-lint (2.12.1)

[error] variable cap has same name as predeclared identifier

(predeclared)